blocks/c-banner/c-banner.js

```diff
@@ -1,8 +1,34 @@
 export default function decorate(block) {
+  const parentSelector = block.closest('.section');
+  const metaData = parentSelector.dataset;
+  const {
+    textColor, backgroundColor, padding, margin,
+  } = metaData;
   const [richTextEl, pictureEl] = [...block.children];
 
+  if (backgroundColor) {
+    parentSelector.style.backgroundColor = backgroundColor;
+  }
+
+  if (textColor) {
+    block.style.textColor = textColor;
+    const elements = block.querySelectorAll('*');
+
+    elements.forEach((element) => {
```

Comment:
Why do we need a forEach? The color property should be inherited from the parent.

Reply:
updated